packages/ui-app/src/Notification.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { BareProps, UnlockI18n } from './types';
+
+import React from 'react';
+
+import Message from 'semantic-ui-react/dist/commonjs/collections/Message/index';
+// import { SemanticSIZES } from 'semantic-ui-react/dist/commonjs/index';
```

Comment:
Remove line, unused code. (If there is a specific reason, indicate it as such with an extra note)